web/src/components/share/invitations/index.tsx

```diff
@@ -202,49 +229,34 @@ const InvitationCard = ({ selectedNetwork, qrCode, qrCodeImage }) => {
       alignContent:'center',
     },
     qrCode: {
-      display: 'flex',
       whiteSpace: 'normal',    
       wordBreak: 'break-all',
-
+      paddingTop: '10px',
+      maxWidth: '100%',
           // Allow text to wrap to the next line
       // hyphens: 'auto',             // Add hyphenation to break long words
-      flexWrap: 'nowrap',
-      position: 'absolute',
-      width: '100%',
-      fontSize:'10px',
-      bottom:'10px',
+      flexWrap: 'wrap',
+      width: '150px',
+      fontSize:'8px',
     },
     networkTitle: {
-      display: 'flex',
-      flexDirection: 'column',
       fontWeight: 'bold',
-      gap: '5px',
+      alignContent: 'center',
+      paddingTop: '0',
     }
   });
 
   return (
     <View style={styles.cardWrapper}>
-      <View style={styles.networkHeader}>
-        <Image
-          style={styles.networkLogo}
-          src={makeImageUrl(selectedNetwork.image)}
-        />
-        <View style={styles.networkTitle}>
-          <Text>{selectedNetwork.name}</Text>
-          <Text>{t('invitation.card')}</Text>
-        </View>
-      </View>
+
       <View style={styles.invitationCard}>
-        <View style={styles.textDiv}>
-          <Text>{t('invitation.infoCard')}</Text>
-          <Text>{t('invitation.name')}</Text>
-          <Text>{t('invitation.contact')}</Text>
-          <Text>{t('invitation.infoQr')}</Text>
+        <View style={styles.networkTitle}>
+          <Text style={styles.networkTitleText}>{selectedNetwork.name}</Text>
         </View>
         <View style={styles.qrDiv}>
           <Image src={qrCodeImage} />
         </View>
-        <Text style={styles.qrCode}>{qrCode}</Text>
+        <Text style={styles.qrCode}>Personal ID: {qrCode}</Text>
```

Comment:
"Personal ID " is very weird string, besides it should be on the translations file :)

Reply:
changed